<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChefForm</class>
 <widget class="QWidget" name="ChefForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="m_authArea">
     <item>
      <widget class="QCheckBox" name="codeEnabledBox">
       <property name="toolTip">
        <string>If checked, users can connect to the server using the current authorization code</string>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Enable Guests</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="authCodeEdit">
       <property name="placeholderText">
        <string>Set guest login code here...</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="tbArea">
       <item>
        <widget class="QPushButton" name="tbSetupButton">
         <property name="text">
          <string>TB Setup</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="talkButton">
         <property name="text">
          <string>TB</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="ChatForm" name="m_chatForm" native="true"/>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="m_actCompArea"/>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="m_actCompArea"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QScrollArea" name="m_channelStripScrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="m_channelStripScrollParent">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>565</width>
        <height>236</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChatForm</class>
   <extends>QWidget</extends>
   <header>chatform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
